src/components/Loading/Loading.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import styles from './Loading.module.scss';
+import classNames from 'classnames/bind';
+import loadingImg from '../../images/loading.gif';
+
+interface Props {
+  loading: boolean;
+  children: () => React.ReactElement;
+}
+
+const cx = classNames.bind(styles);
+
+const Loading: React.FC<Props> = ({ loading, children }) => {
+  return (
+    loading ?
+      (
+        <>
+          <div className={cx('loading-img')}>
+            <img src={loadingImg} alt="" />
+          </div>
+          {children()}
+        </>
+      ) :
+      (
+        children()
+      )
+  )
+};
```

Comment:
nit 하긴 한데 이런식으로도 간소화 할 수 있을 것 같아요.

```suggestion
const Loading: React.FC<Props> = ({ loading, children }) => (
<>
  {
    loading && ( 
      <div className={cx('loading-img')}>
        <img src={loadingImg} alt="" />
      </div>
    )
  }
  {children}
</>
);
```

Reply:
loading시에 children이 꼭 있어야 하는건가요?

Reply:
@eastroots92 님 말씀대로 줄일수 있네요! 감사합니다.

Reply:
loading시에 로딩이미지에 투명도를 주어 뒤가 반투명하게 보이도록 해서 loading시에도 children컴포넌트가 렌더링 되도록 했습니다. 사실 loading시엔 children컴포넌트가 렌더링 될 필요가 없긴 하지요..